### ISSUE TITLE
Get info from the studyPar if the DICOMs are anonymized

### ISSUE THREAD

**Issue by BeatrizPadrela** [ExploreASL/ExploreASL#739]
### Description

If we have DICOMs that are all anonymized (including the Manufacturer name for example - yes, this happens with the twins data), we can write it in the studyPar so that the import module can recognize it.

### Requirements

Change the import functions that recognize the Manufacturer from the DICOMs and add an option to read it from the studyPar

### How to test

Optional: insert description about how to test the code changes here

### Release notes

Required: summarize the changes for the release notes here

**Comment by MichaelStritt** [comment]
Yep, so we need to make sure that the import doesn't crash if there is no Manufacturer in the DICOMs & that it is read from the studyPar or dataPar in that case (if it exists there). If it doesn't exist in either or, we should probably still raise an error.

**Comment by BeatrizPadrela** [comment]
Thanks!! Will check this when the processing of the EPAD data finished (is running on develop)

**Comment by BeatrizPadrela** [comment]
Actually, I will gitclone ExploreASL to another folder so I can use this branch *sunglasses smile*

**Comment by BeatrizPadrela** [comment]
It works!! Just gives warnings for the missing fields

**Comment by BeatrizPadrela** [comment]
works with and without "Manufacturer" field in the Studypar!
